docs/src/5-prestashop-billing/README.md

```diff
@@ -306,6 +306,163 @@ To do so:
   window.psBilling.initializeInvoiceList(window.psBillingContext.context, '#ps-billing-invoice');
   ```
 
+
+## (Optional) Handle plan selection by yourself
+
+The billing funnel can handle for you the plan selection, but sometimes you should prefer to present your plan by yourself to be more precise about your offer.
```

Comment:
```suggestion
The billing funnel can handle for you the plan selection, but sometimes you may prefer to present your plan by yourself to be more precise about your offer.
```
should dans ce contexte c'est une recommendation très appuyée, mais pas une obligation, or ici on veut dire au partner que s'il le souhaite, il a cette possibilité.